modules/imgproc/src/peronamalik.cpp

```diff
@@ -0,0 +1,89 @@
+
+#include "precomp.hpp"
+#include "opencl_kernels_imgproc.hpp"
+#include "cvconfig.h"
+
+#include <iostream>
+using namespace std;
```

Comment:
Look like it is not used here.

BTW, it is better to place "using namespace" under "namespace cv" to prevent global namespace corruption (for single .cpp file there is no impact actually).